.github/workflows/test.yml

```diff
@@ -0,0 +1,27 @@
+name: Tests for lwt_react, lwt_ppx, lwt_luv and lwt
+on: [push, pull_request]
+jobs:
+  test:
+    strategy:
+      matrix:
+        operating-system: [macos-latest, ubuntu-latest, windows-latest]
+        ocaml-version: [4.11.0, 4.10.0, 4.09.1]
```

Comment:
```suggestion
        ocaml-version: [4.11.1, 4.10.2, 4.09.1]
```

Reply:
Thanks! I should be using OCaml Version in `opam-github-workflow` https://github.com/patricoferris/opam-github-workflow/issues/8

Reply:
I recommend quoting the versions, in some cases they might interpreted as float instead of strings.

Reply:
Thanks for the good suggestion, have added both in latest commit and removed `lwt_luv` 👍  I'll have to work out how to force quotation in the afore-mentioned tool for generating these workflow files.